packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -1382,10 +1383,28 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
     return imageVolume.imageIds;
   };
 
-  abstract getCurrentImageIdIndex(): number;
-
   abstract getCurrentImageId(): string;
 
+  public getTargetId(specifier: TargetSpecifier = {}): string {
+    let { volumeId, sliceIndex } = specifier;
+    if (!volumeId) {
+      const actorEntries = this.getActors();
+      if (!actorEntries) {
+        return;
+      }
+      // find the first image actor of instance type vtkVolume
+      volumeId = actorEntries.find(
+        (actorEntry) => actorEntry.actor.getClassName() === 'vtkVolume'
+      )?.uid;
+    }
+
+    sliceIndex ??= this.getCurrentImageIdIndex();
+    const { viewPlaneNormal, focalPoint } = this.getCamera();
+    return `volumeId:${volumeId}?sliceIndex=${sliceIndex}&viewPlaneNormal=${viewPlaneNormal.join(
```

Comment:
let's include viewUp as well

Reply:
I'm not convinced about that one - the viewUp isn't relevant for whether the annotation applies or not.  The normal and the focal point are relevant, and perhaps the pan, zoom and thickness, but I don't see how the viewUp, rotation, flip matter.

Reply:
ok